nearuplib/nodelib.py

```diff
@@ -314,7 +314,7 @@ def run_nodocker(home_dir, binary_path, boot_nodes, verbose, chain_id):
     proc = run_binary(os.path.join(binary_path, 'near'), home_dir, 'run', verbose=verbose,
                       boot_nodes=boot_nodes, output=os.path.join(LOGS_FOLDER, chain_id))
     proc_name = proc_name_from_pid(proc.pid)
-    print(proc.pid, "|", proc_name, file=pid_fd)
+    print(proc.pid, "|", proc_name, "|", chain_id, file=pid_fd)
```

Comment:
What is this solving right now? if I understand correctly this will be used to solve #34 in future PR, is it?

Reply:
It already resolves #34. How this works: we always write the chain id into that file and when we call `stop_native` the function looks for the chain id in the pid file and now it is guaranteed that it will find the correct chain id.